int16 x
int16 y
int16 w
int16 h
float32 score
